src/redux/stockDetails/stockDetailsSlice.js

```diff
@@ -0,0 +1,63 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const key = '?apikey=9eb223102d06629a659db2e7283983fd';
+const stockDetailsUrl = 'https://financialmodelingprep.com/api/v3/quote/';
+
+const initialState = {
+  stockDetails: [],
+  isLoading: false,
+  isError: false,
+  fetched: false,
+};
+
+export const fetchStockDetails = createAsyncThunk('stockDetails/fetchStockDetails', async (symbol) => {
+  try {
+    const response = await stockDetailsUrl + symbol + key;
```

Comment:
- [ ] @imranhossainemi kindly check on your method for fetching the stock details, currently, the page is not loading any data, showing loading data as shown below,(asked for this change as the app was working correctly before you made the new changes after the first review)
![image](https://user-images.githubusercontent.com/61408860/234972937-1f0c3b11-82a9-41b0-9758-2c1a0430c439.png)